islandora.api.php

```diff
@@ -623,7 +623,12 @@ function hook_islandora_overview_object_alter(AbstractObject &$object, &$output)
  * Content model specific.
  */
 function hook_cmodel_pid_islandora_overview_object_alter(AbstractObject &$object, &$output) {
-  $output = $output . drupal_render(drupal_get_form('some_form', $object));
+  $view = views_embed_view('usage_collection');
+  $form['islandora_manage_overview_object'] = array(
+    '#type' => 'item',
+    '#markup' => $view,
+  );
+  $output = $output . $form . drupal_render(drupal_get_form('some_form', $object)));
```

Comment:
I think that instead of this, you probably want to implement this hook in the [`islandora_usage_stats`](https://github.com/discoverygarden/islandora_usage_stats) module? If that is the case, [`islandora_usage_stats.module`](https://github.com/knjackson30974/islandora_usage_stats/blob/Kylea/7.x/islandora_usage_stats.module) should contain something like:
```php
function islandora_usage_stats_islandora_collectioncmodel_islandora_overview_object_alter(AbstractObject &$object, &$output) {
  // The key here doesn't really matter... Should just try to make sure it's unique (module prefixing helps this).
  $output['islandora_usage_stats_collection_stats'] = array(
    '#type' => 'item',
    '#markup' => views_embed_view('usage_collection'),
  );
}
```